### DISCUSSION TITLE
为什么我这个翻译不了啊，是代理的问题不

### DISCUSSION THREAD

**Discussion by Virnnar** [pot-app/pot-desktop#316] [Q&A]
<img width="265" alt="image" src="https://github.com/pot-app/pot-desktop/assets/109610268/3f0fb9a5-29ef-4d44-83c1-63da4331e058">

**Comment by Pylogmon** [comment]
确实是网络问题，大概率和代理设置有关，你的代理设置怎么填的，不需要的话一般空着就行。

**Reply by Virnnar** [reply]
是空着的、刚刚试了一下http://127.0.0.1:7890 也是这样的问题

**Reply by Pylogmon** [reply]
只有deepl有问题吗，还是所有接口都这样。

**Reply by Virnnar** [reply]
目前bing和DeepL都是 :(

**Reply by Pylogmon** [reply]
留空之后点应用，然后重启软件或重启电脑试试

**Reply by Virnnar** [reply]
好的好的我试一下

**Reply by Virnnar** [reply]
更新Clash后问题解决了:)，但是问题偶尔还是会出现（基本上不会  感谢<333
